Source/Scene/Model.js

```diff
@@ -1533,6 +1533,17 @@ define([
 
     ///////////////////////////////////////////////////////////////////////////
 
+    function loadGLExtensions(model, context) {
```

Comment:
There's no need for this and this level of the Cesium stack should avoid making direct GL calls, see http://cesiumjs.org/2015/05/15/Graphics-Tech-in-Cesium-Architecture/

The uint32 extension is initialized here: https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Renderer/Context.js#L277

`Model` should check [Context.elementIndexUint](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Renderer/Context.js#L553-L557) and throw a `RuntimeError` if the model wants to use the uint32 extension, but the system does not support it.